assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/cl/install.yaml.tmpl

```
@@ -32,7 +32,11 @@ storage:
           #!/bin/bash -ex
           curl --retry 10 "${ignition_endpoint}?{{.request.raw_query}}&os=installed" -o ignition.json
           flatcar-install \
+            %{~ if install_to_smallest_partition ~}
```

```suggestion
            %{~ if install_to_smallest_disk ~}
```
It's installing to the full disk

---

ah, I see